### PR TITLE
Created repositore service controller for store

### PR DESCRIPTION
**Pull request by grigveronika**

### CODE REVIEW

File: src/main/java/internship/mbicycle/storify/controller/StoreController.java [hunk 1]
```diff
@@ -1,16 +1,49 @@
...
 
+import java.util.List;
+
 @RestController
 @RequiredArgsConstructor
 @RequestMapping("/stores")
 public class StoreController {
     private final StoreService storeService;
 
+    @GetMapping("/findAll/{profileId}")
```
Comment (Estrin-MBicycle) [inline src/main/java/internship/mbicycle/storify/controller/StoreController.java:23]: read endpoint naming rules

**Review by Estrin-MBicycle — CHANGES_REQUESTED** [review]

File: src/main/java/internship/mbicycle/storify/controller/StoreController.java (hunk 1 above)
Comment (grigveronika) [reply src/main/java/internship/mbicycle/storify/controller/StoreController.java]: changed names of address resources in Storecontroller

File: src/main/java/internship/mbicycle/storify/controller/StoreController.java [hunk 2]
```diff
@@ -20,28 +20,28 @@
 public class StoreController {
     private final StoreService storeService;
 
-    @GetMapping("/findAll/{profileId}")
+    @GetMapping("/{profileId}")
     public List<StoreDTO> findAllStores(@PathVariable Long profileId) {
```
Comment (Estrin-MBicycle) [inline src/main/java/internship/mbicycle/storify/controller/StoreController.java:24]: getAllStores

File: src/main/java/internship/mbicycle/storify/controller/StoreController.java [hunk 3]
```diff
@@ -20,28 +20,28 @@
...
         return storeService.findStoresByProfileId(profileId);
     }
 
-    @GetMapping("/findOneStore/{id}/{profileId}")
+    @GetMapping("/{id}/{profileId}")
     public StoreDTO findOneStore(@PathVariable Long id, @PathVariable Long profileId) {
```
Comment (Estrin-MBicycle) [inline src/main/java/internship/mbicycle/storify/controller/StoreController.java:29]: findStore

File: src/main/java/internship/mbicycle/storify/controller/StoreController.java [hunk 4]
```diff
@@ -20,28 +20,28 @@
...
         return storeService.findStoresByIdAndProfileId(id, profileId);
     }
 
-    @PostMapping("/saveStore")
+    @PostMapping()
     public StoreDTO saveUsers(@RequestBody StoreDTO storeDTO)  {
         return storeService.saveStore(storeDTO);
```
Comment (Estrin-MBicycle) [inline src/main/java/internship/mbicycle/storify/controller/StoreController.java:35]: saveStore

File: src/main/java/internship/mbicycle/storify/controller/StoreController.java [hunk 5]
```diff
@@ -20,28 +20,28 @@
...
     }
 
-    @DeleteMapping("/deleteAllStores/{profileId}")
+    @DeleteMapping("/{profileId}")
     public ResponseEntity<Void> deleteUsers(@PathVariable Long profileId) {
```
Comment (Estrin-MBicycle) [inline src/main/java/internship/mbicycle/storify/controller/StoreController.java:39]: deleteUserProfile (removeUserProfile)
use ? instaed of void

File: src/main/java/internship/mbicycle/storify/controller/StoreController.java [hunk 6]
```diff
@@ -20,28 +20,28 @@
...
         storeService.deleteAllByProfileId(profileId);
         return ResponseEntity.ok().build();
     }
 
-    @DeleteMapping("/deleteOneStore/{id}/{profileId}")
+    @DeleteMapping("/{id}/{profileId}")
     public ResponseEntity<Void> deleteUsers(@PathVariable Long id, @PathVariable Long profileId) {
```
Comment (Estrin-MBicycle) [inline src/main/java/internship/mbicycle/storify/controller/StoreController.java:45]: deleteStore (removeStore)

**Review by Estrin-MBicycle — CHANGES_REQUESTED** [review]

File: src/main/java/internship/mbicycle/storify/controller/StoreController.java (hunk 2 above)
Comment (grigveronika) [reply src/main/java/internship/mbicycle/storify/controller/StoreController.java]: done

File: src/main/java/internship/mbicycle/storify/controller/StoreController.java (hunk 4 above)
Comment (grigveronika) [reply src/main/java/internship/mbicycle/storify/controller/StoreController.java:35]: done

File: src/main/java/internship/mbicycle/storify/controller/StoreController.java (hunk 6 above)
Comment (grigveronika) [reply src/main/java/internship/mbicycle/storify/controller/StoreController.java]: done

File: src/main/java/internship/mbicycle/storify/controller/StoreController.java (hunk 3 above)
Comment (grigveronika) [reply src/main/java/internship/mbicycle/storify/controller/StoreController.java]: done

**Pull request merged**